script/Arcueid/Day 3/03_00_ARC03_1.txt

```diff
@@ -46,17 +50,20 @@ These 'lines' appear equally on everything I can see.
 [sha:e90366f90b21da1ba0ade1b5c14ada03f400e1fd]{
 -- Page 3, Offset 3954.
 -- あらゆるものは『壊れてしまう』という運命を内包しているの。カタチがある以上、こればっかりは逃れようのない前提でしょう？」
-All things are destined to break. As long as it has a form, it can't escape that fate, right?"
+Everything that exists is destined to break eventually.
```

Comment:
I prefer "all things are destined to break". "that exist" is unnecessary added connotation (that there are things that 'do not exist')